compiler/rustc_middle/src/mir/visit.rs

```diff
@@ -1320,7 +1320,7 @@ pub enum NonUseContext {
     /// Ending a storage live range.
     StorageDead,
     /// User type annotation assertions for NLL.
-    AscribeUserTy,
+    AscribeUserTy(Variance),
```

Comment:
```suggestion
    AscribeUserTy(ty::Variance),
```
For uniformity in the file.